sklearn/utils/estimator_checks.py

```diff
@@ -2974,7 +2974,9 @@ def check_estimator_sparse_dense(name, estimator_orig,
     else:
         tags = estimator_orig._get_tags()
         centers = 2 if tags["binary_only"] else None
-        X, y = make_blobs(random_state=rng, cluster_std=0.5, centers=centers)
+        X, y = make_blobs(n_samples=10, n_features=2, random_state=rng,
```

Comment:
I simplified the data here similarly to before, because there were some numerical uncertainties with the Nystrom method (particularly with almost zero elements: the relative error between two numbers can be quite high but both number can be of order 1e-15 for instance, so they are clearly supposed to be zeros). With a simpler dataset that will lead to less operations (because less samples, less features, and less decimals), the uncertainties disappeared on my machine